x-pack/plugins/search_connectors/server/routes/connectors_routes.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CONNECTOR_DEFINITIONS } from '@kbn/search-connectors';
+import { RouteDependencies } from '../plugin';
+
+export const registerConnectorsRoutes = ({ http, router }: RouteDependencies) => {
+  router.get(
+    {
+      path: '/internal/search_connectors/connector_types',
```

Comment:
Are we still using this route? it looks like the search plugins are reading these types from the plugin start object and it's getting them from common.

Reply:
Oh huh I thought I'd deleted this, removing it!